lib/options_validator.js

```diff
@@ -89,6 +95,17 @@ function validationFunction(optionsSchema, providedOptions, validationOptions) {
       const deprecationMessage = `option [${optionName}] is deprecated and will be removed in a later version.`;
       emitDeprecationWarning(deprecationMessage);
     }
+
+    if (overrideOptions && overrideOptions[optionName] != null) {
```

Comment:
we seem to have lost my last comment, but to reiterate: if we intend these overrides to actually apply in the `validationLevel: none` case, the overrides need to apply first above. You will also need to detect if there is a `default` attempting to overwrite a provided `override` option.